rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -32,7 +32,10 @@ This document does not define how to transport data to other participants in a d
 
 * **Application data**: data produced and consumed by the application that uses the formats described by this RFC.
 * **Transaction**: node on the graph consisting of application data and metadata like signatures and references to other transactions published on a distributed network.
-* **Root transaction** the first transaction published on a network.
+* **Root transaction**: the first transaction published on a network.
+* **JWS payload**: The payload as defined by [RFC7515](https://tools.ietf.org/html/rfc7515#section-3).
+* **Transaction payload**: The actual data in the transaction.
```

Comment:
maybe we should rename `transaction payload` to `transaction content`?

Reply:
I'll do a find replace, see if it works.

Reply:
changed to `content`